sql/versioned/V2__add_number_of_pets_column_to_customers.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE customers
+    ADD COLUMN number_of_pets bigint
```

Comment:
Do we really need this to be a `bigint`?

Reply:
Yes because reasons